app/recipe/views/recipe.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework import viewsets
+from rest_framework.filters import OrderingFilter
+from django_filters.rest_framework import DjangoFilterBackend
+
+
+from recipe.models import Recipe
+from recipe.serializers import RecipeSerializer, RecipeDetailSerializer
+from recipe.filters import RecipeFilter
+
+
+class RecipeViewSet(viewsets.ModelViewSet):
```

Comment:
As with models, you don't need either one separate file per view class, but it is OK!

Reply:
Same as above. Thoughts?